CSS/header.css

```diff
@@ -1,67 +1,67 @@
 .header {
-    position: fixed;
-    top: 0;
-    left: 0;
-    right: 0;
-    background: #fff;
-    border-bottom: 1px solid #AAAAAA;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  background: #fff;
+  border-bottom: 1px solid #AAAAAA;
```

Comment:
better to keep ony way of puttung colors
#fff and #aaa
 or
 #FFF and #AAA 
or 
#ffffff and #aaaaaa
or 
#FFFFFF and #AAAAAA